### ISSUE TITLE
[DETECTION] ADVobfuscator in PGSharp

### ISSUE THREAD

**Issue by enovella** [rednaga/APKiD#302]
APK: https://www.pgsharp.com/

```sh
[edu@xps arm64-v8a] >  r2 libadventuresync.so 
Warning: run r2 with -e bin.cache=true to fix relocations in disassembly
 -- Safety third
[0x00141bf0]> izzq~+advobfuscator
0x9b506 364 363 _ZN8andrivet13ADVobfuscator11MetaString4ILi2ELi86ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54ELi55ELi56ELi57ELi58EEEEE7decryptEv
0xb4090 359 358 _ZN8andrivet13ADVobfuscator11MetaString4ILi2ELi86ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54ELi55ELi56ELi57EEEEE7decryptEv
0xd7d78 359 358 _ZN8andrivet13ADVobfuscator11MetaString4ILi2ELi90ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54ELi55ELi56ELi57EEEEE7decryptEv
0xda1ca 344 343 _ZN8andrivet13ADVobfuscator11MetaString4ILi2ELi62ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54EEEEE7decryptEv
0xe04f8 358 357 _ZN8andrivet13ADVobfuscator11MetaString4ILi2ELi8ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54ELi55ELi56ELi57EEEEE7decryptEv
0xe1615 359 358 _ZN8andrivet13ADVobfuscator11MetaString4ILi2ELi57ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54ELi55ELi56ELi57EEEEE7decryptEv
[0x00141bf0]>
```

```sh
apkid /tmp/pgs1.67.4.apk 
[+] APKiD 2.1.3 :: from RedNaga :: rednaga.io
[*] /tmp/pgs1.67.4.apk!classes.dex
 |-> anti_vm : Build.FINGERPRINT check, Build.MANUFACTURER check, Build.MODEL check, Build.PRODUCT check, SIM operator check, network operator name check, possible VM check, ro.kernel.qemu check
 |-> compiler : dexlib 2.x
[*] /tmp/pgs1.67.4.apk!classes2.dex
 |-> anti_debug : Debug.isDebuggerConnected() check
 |-> anti_vm : Build.FINGERPRINT check, Build.HARDWARE check, Build.MANUFACTURER check, Build.MODEL check, Build.PRODUCT check, Build.TAGS check, network operator name check, possible VM check
 |-> compiler : dexlib 2.x
[*] /tmp/pgs1.67.4.apk!classes3.dex
 |-> anti_vm : Build.FINGERPRINT check, Build.MANUFACTURER check
 |-> compiler : dexlib 2.x
[*] /tmp/pgs1.67.4.apk!lib/arm64-v8a/libardk_client_platform.so
 |-> obfuscator : Arxan
[*] /tmp/pgs1.67.4.apk!lib/arm64-v8a/libNianticLabsPlugin.so
 |-> obfuscator : Arxan
[*] /tmp/pgs1.67.4.apk!lib/armeabi-v7a/libardk_client_platform.so
 |-> obfuscator : Arxan
[*] /tmp/pgs1.67.4.apk!lib/armeabi-v7a/libNianticLabsPlugin.so
 |-> obfuscator : Arxan
[*] /tmp/pgs1.67.4.apk!lib/arm64-v8a/libmain.so
 |-> obfuscator : Obfuscator-LLVM version unknown (string encryption)
[*] /tmp/pgs1.67.4.apk!lib/armeabi-v7a/libmain.so
 |-> obfuscator : Obfuscator-LLVM version unknown (string encryption)
```

**Comment by apkunpacker** [comment]
sample - 
```sh
9220d999e96aa20d10ce936380853e9e
```
```sh
$ r2 libnatives.so
Warning: run r2 with -e bin.cache=true to fix relocations in disassembly
 -- Coffee time!
[0x0000fd1c]> izzq~+advobfuscator
0x4201 164 163 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc104ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18EEEEE7decryptEv
0x42a5 103 102 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc107ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x430c 99 98 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc109ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4EEEEE7decryptEv
0x436f 234 233 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc109ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32EEEEE7decryptEv
0x4459 398 397 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc10ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54ELi55ELi56ELi57ELi58ELi59ELi60ELi61ELi62ELi63ELi64ELi65EEEEE7decryptEv
0x45e7 91 90 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc110ENS0_7IndexesIJLi0ELi1ELi2EEEEE7decryptEv
0x4642 91 90 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc115ENS0_7IndexesIJLi0ELi1ELi2EEEEE7decryptEv
0x469d 254 253 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc124ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36EEEEE7decryptEv
0x479b 143 142 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc16ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14EEEEE7decryptEv
0x482a 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc17ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x4890 183 182 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc18ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22EEEEE7decryptEv
0x4947 94 93 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc19ENS0_7IndexesIJLi0ELi1ELi2ELi3EEEEE7decryptEv
0x49a5 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc27ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x4a0b 114 113 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc28ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8EEEEE7decryptEv
0x4a7d 133 132 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc28ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12EEEEE7decryptEv
0x4b02 198 197 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc43ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25EEEEE7decryptEv
0x4bc8 98 97 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc58ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4EEEEE7decryptEv
0x4c2a 183 182 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc69ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22EEEEE7decryptEv
0x4ce1 158 157 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc71ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17EEEEE7decryptEv
0x4d7f 178 177 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc79ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21EEEEE7decryptEv
0x4e31 298 297 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc84ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45EEEEE7decryptEv
0x4f5b 273 272 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc85ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40EEEEE7decryptEv
0x506c 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi0ELc92ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x50d2 95 94 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc115ENS0_7IndexesIJLi0ELi1ELi2ELi3EEEEE7decryptEv
0x5131 103 102 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc118ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x5198 169 168 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc122ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19EEEEE7decryptEv
0x5241 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc14ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x52a7 343 342 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc15ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40ELi41ELi42ELi43ELi44ELi45ELi46ELi47ELi48ELi49ELi50ELi51ELi52ELi53ELi54EEEEE7decryptEv
0x53fe 148 147 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc16ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15EEEEE7decryptEv
0x5492 106 105 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc25ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6EEEEE7decryptEv
0x54fc 178 177 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc46ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21EEEEE7decryptEv
0x55ae 98 97 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc47ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4EEEEE7decryptEv
0x5610 258 257 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc48ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37EEEEE7decryptEv
0x5712 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc56ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x5778 138 137 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc60ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13EEEEE7decryptEv
0x5802 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc63ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x5868 148 147 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc63ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15EEEEE7decryptEv
0x58fc 123 122 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc73ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10EEEEE7decryptEv
0x5977 137 136 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc7ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13EEEEE7decryptEv
0x5a00 123 122 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc82ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10EEEEE7decryptEv
0x5a7b 106 105 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc86ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6EEEEE7decryptEv
0x5ae5 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi1ELc93ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x5b4b 174 173 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc107ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20EEEEE7decryptEv
0x5bf9 164 163 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc116ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18EEEEE7decryptEv
0x5c9d 169 168 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc119ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19EEEEE7decryptEv
0x5d46 91 90 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc121ENS0_7IndexesIJLi0ELi1ELi2EEEEE7decryptEv
0x5da1 123 122 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc13ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10EEEEE7decryptEv
0x5e1c 188 187 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc19ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23EEEEE7decryptEv
0x5ed8 158 157 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc32ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17EEEEE7decryptEv
0x5f76 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc43ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x5fdc 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc54ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x6042 273 272 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc55ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30ELi31ELi32ELi33ELi34ELi35ELi36ELi37ELi38ELi39ELi40EEEEE7decryptEv
0x6153 128 127 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc69ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11EEEEE7decryptEv
0x61d3 110 109 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc72ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7EEEEE7decryptEv
0x6241 148 147 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc86ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15EEEEE7decryptEv
0x62d5 94 93 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc87ENS0_7IndexesIJLi0ELi1ELi2ELi3EEEEE7decryptEv
0x6333 158 157 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc87ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17EEEEE7decryptEv
0x63d1 98 97 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc93ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4EEEEE7decryptEv
0x6433 223 222 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc94ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17ELi18ELi19ELi20ELi21ELi22ELi23ELi24ELi25ELi26ELi27ELi28ELi29ELi30EEEEE7decryptEv
0x6512 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc98ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x6578 102 101 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc99ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5EEEEE7decryptEv
0x65de 157 156 _ZN8andrivet13ADVobfuscator10MetaStringILi2ELc9ENS0_7IndexesIJLi0ELi1ELi2ELi3ELi4ELi5ELi6ELi7ELi8ELi9ELi10ELi11ELi12ELi13ELi14ELi15ELi16ELi17EEEEE7decryptEv
```

**Comment by apkunpacker** [comment]
```sh
e1ecb15518cb719134623c530dc243ea
c9448692ec01599a723dc1e3bb80b816
94ba2ea86ac4d865eaa72d19992b44ce
36008164a142a3f3cd9b6967e8b3f801
73f449e26b260a5bfca1a55c18ee59e1
fd4339810094222db1ed03175a75b92d
```

**Comment by enovella** [comment]
@apkunpacker can you verify your samples?
